### PR TITLE
Xmr btc rendezvous release

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: blog/2021-07-08-xmr-btc-project-hibernation.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+In summary, we have pretty much delivered exactly what we wanted:
+
+A somewhat usable tool that allows users to trustlessly trade the XMR/BTC pair.
+
+## What's next
+
+We are still very keen on offering the missing feature of selling XMR for BTC.
+Outside of that, we also had some ideas on what _could_ be built on top of or with our tool:
+
+- Swaps completely in the browser:
```
Comment (bonomat) [inline blog/2021-07-08-xmr-btc-project-hibernation.md:72]: I'd add to the top of the list the swap in the other direction and saying that we will very likely integrate this as soon as it's possible.

File: blog/2021-07-08-xmr-btc-project-hibernation.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  Whilst sufficient for expert users and to showcase that it works, a proper GUI would be a lot more user-friendly.
+  Fortunately, the CLI is completely non-interactive.
+  If anyone wants to build a GUI, the recommended way would be to start the CLI for the user in the background and process the logs delivered on stderr.
+  An adventurous developer could also try to build a GUI directly in Rust and integrate on a library level.
+- A mobile app.
+  We have very little experience in our team when it comes to writing mobile apps which is why we didn't focus on this at all.
+  For all we know, Rust can target Android as well as iOS which means there should be a way of getting this onto mobile!
+
+This list is by no means exhaustive and we would love for others to jump onto some of these ideas.
+Whilst we will not be pushing any of those forward ourselves, we are more than happy to assist by participating and steering [discussions](https://github.com/comit-network/xmr-btc-swap/discussions) and reviewing pull requests.
```
Comment (bonomat) [inline blog/2021-07-08-xmr-btc-project-hibernation.md:89]: You could thank the community for feedback and contributions and maybe even highlight the contributions we got.

**Review by bonomat — APPROVED** [review]
Concept ACK. Didn't check for typos/grammer.

File: blog/2021-07-08-xmr-btc-project-hibernation.md (hunk 2 above)
Comment (thomaseizinger) [reply blog/2021-07-08-xmr-btc-project-hibernation.md:89]: Nice idea, should do that yes!

File: blog/2021-07-08-xmr-btc-project-hibernation.md (hunk 1 above)
Comment (thomaseizinger) [reply blog/2021-07-08-xmr-btc-project-hibernation.md:72]: That is what I tried to say with the very first line:

>  We are still very keen on offering the missing feature of selling XMR for BTC.

**Review by thomaseizinger — APPROVED** [review]

File: blog/2021-07-08-xmr-btc-project-hibernation.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+- A mobile app.
+  We have very little experience in our team when it comes to writing mobile apps which is why we didn't focus on this at all.
+  For all we know, Rust can target Android as well as iOS which means there should be a way of getting this onto mobile!
+
+This list is by no means exhaustive, and we would love for others to jump onto some of these ideas.
+Whilst we will not be pushing any of those forward ourselves, we are more than happy to assist by participating and steering [discussions](https://github.com/comit-network/xmr-btc-swap/discussions) and reviewing pull requests.
+
+Thanks to the community for the contributions so far:
+
+- Documentation ([#609](https://github.com/comit-network/xmr-btc-swap/pull/606), [#614](https://github.com/comit-network/xmr-btc-swap/pull/614), [#533](https://github.com/comit-network/xmr-btc-swap/pull/533))
```
Comment (thomaseizinger) [inline blog/2021-07-08-xmr-btc-project-hibernation.md:108]:
```suggestion
- Documentation ([#609](https://github.com/comit-network/xmr-btc-swap/pull/609), [#614](https://github.com/comit-network/xmr-btc-swap/pull/614), [#533](https://github.com/comit-network/xmr-btc-swap/pull/533))
```

**Review by thomaseizinger — CHANGES_REQUESTED** [review]

**Pull request merged**